core-plugins/widgets/XMLReader-batchsource.json

```diff
@@ -67,7 +67,6 @@
           "label": "Table Data Expiry Period (Days)",
```

Comment:
the widget type should be a textbox. there is a bug with the number widet-type: https://issues.cask.co/browse/HYDRATOR-1187